tests/fast-integration/test/fixtures/jaeger/jaeger.yaml

```diff
@@ -0,0 +1,307 @@
+    app.kubernetes.io/instance: tracing-jaeger
+    app.kubernetes.io/managed-by: jaeger-operator
+    app.kubernetes.io/name: tracing-jaeger
+    app.kubernetes.io/part-of: jaeger
+  name: tracing-jaeger
+  namespace: kyma-system
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: jaeger
+      app.kubernetes.io/component: all-in-one
+      app.kubernetes.io/instance: tracing-jaeger
+      app.kubernetes.io/managed-by: jaeger-operator
+      app.kubernetes.io/name: tracing-jaeger
+      app.kubernetes.io/part-of: jaeger
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      annotations:
+        linkerd.io/inject: disabled
+        prometheus.io/port: "14269"
+        prometheus.io/scrape: "true"
+        sidecar.istio.io/inject: "true"
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+      labels:
+        app: jaeger
+        app.kubernetes.io/component: all-in-one
+        app.kubernetes.io/instance: tracing-jaeger
+        app.kubernetes.io/managed-by: jaeger-operator
+        app.kubernetes.io/name: tracing-jaeger
+        app.kubernetes.io/part-of: jaeger
+    spec:
+      containers:
+      - args:
+        - --collector.otlp.enabled=true
+        - --log-level=info
+        - --memory.max-traces=10000
+        - --query.ui-config=/etc/config/ui.json
+        - --sampling.strategies-file=/etc/jaeger/sampling/sampling.json
+        env:
+        - name: SPAN_STORAGE_TYPE
+          value: memory
+        - name: METRICS_STORAGE_TYPE
+        - name: COLLECTOR_ZIPKIN_HOST_PORT
+          value: :9411
+        - name: JAEGER_DISABLED
+          value: "false"
+        image: eu.gcr.io/kyma-project/external/jaegertracing/all-in-one:1.37.0
+        imagePullPolicy: IfNotPresent
+        livenessProbe:
+          failureThreshold: 5
+          httpGet:
+            path: /
+            port: 14269
+            scheme: HTTP
+          initialDelaySeconds: 5
+          periodSeconds: 15
+          successThreshold: 1
+          timeoutSeconds: 1
+        name: jaeger
+        ports:
+        - containerPort: 5775
+          name: zk-compact-trft
+          protocol: UDP
+        - containerPort: 5778
+          name: config-rest
+          protocol: TCP
+        - containerPort: 6831
+          name: jg-compact-trft
+          protocol: UDP
+        - containerPort: 6832
+          name: jg-binary-trft
+          protocol: UDP
+        - containerPort: 9411
+          name: zipkin
+          protocol: TCP
+        - containerPort: 14267
+          name: c-tchan-trft
+          protocol: TCP
+        - containerPort: 14268
+          name: c-binary-trft
+          protocol: TCP
+        - containerPort: 16686
+          name: query
+          protocol: TCP
+        - containerPort: 14269
+          name: admin-http
+          protocol: TCP
+        - containerPort: 14250
+          name: grpc
+          protocol: TCP
+        - containerPort: 4317
```

Comment:
You can remove all ports except of the two otlp ports, will make the setup simpler